clamd.go

```diff
@@ -377,14 +377,16 @@ func (c *Client) streamCmd(tc *textproto.Conn, cmd protocol.Command, f io.Reader
 
 	fmt.Fprintf(tc.W, "n%s\n", cmd)
 	b := make([]byte, 4)
-	for {
+	eof := false
+	for !eof {
 		buf := make([]byte, ChunkSize)
 		if n, err = f.Read(buf); err != nil {
 			if err == io.EOF {
```

Comment:
No need for the two branches, check for != io.EOF and return here

Reply:
Done; although I am not sure it increased the readability. That's likely subjective, though.